package/libzypp.changes

```diff
@@ -27,6 +27,11 @@ Mon May  6 08:28:32 UTC 2024 - Dirk Müller <dmueller@suse.com>
 - add one more missing export for PackageKit
 - version 17.33.2
 
+-------------------------------------------------------------------
+Tue May 14 16:17:41 UTC 2024 - Dirk Müller <dmueller@suse.com>
+
+- don't require libproxy1 on tumbleweed, it is optional now
+
```

Comment:
Need to move that part to the top so its in chronological order. Rebase missing maybe?